as-controller-board-status/res/configuration.toml

```diff
@@ -78,7 +65,7 @@ Type="edgex-messagebus"
   DeviceName = "ds-controller-board"
   MaxTemperatureThreshold = "83"
   MinTemperatureThreshold = "10"
-  MQTTEndpoint = "http://localhost:48082/api/v1/device/name/Inference-MQTT-device/command/inferenceDoorStatus" # TODO: update ds-inference-mock to use a different command and device name
+  MQTTEndpoint = "http://localhost:48082/api/v2/device/name/Inference-MQTT-device/command/inferenceDoorStatus" # TODO: update ds-inference-mock to use a different command and device name
```

Comment:
@brian-intel , do we need this TODO? Seems stale....

Reply:
This is out of date and the comment can be removed.